siad/daemon.go

```diff
@@ -189,7 +189,7 @@ func startDaemon(config Config) (err error) {
 func startDaemonCmd(cmd *cobra.Command, _ []string) {
 	// Create the profiling directory if profiling is enabled.
 	if globalConfig.Siad.Profile {
-		go profile.StartContinuousProfile(globalConfig.Siad.ProfileDir)
+		go profile.StartContinuousProfile(globalConfig.Siad.ProfileDir, false, false)
```

Comment:
by default the profiler won't grab anything. I'm almost thinking that we shouldn't have this be a flag, and just have it be a build tag thing. If you build with the 'profile' build tag, you'll get a profiler on port 10501, I don't see why you shouldn't also get the continuous profiling stuff and then the whole profile package can just be left out of the normal binaries.